src/DDTrace/OpenTelemetry/SpanBuilder.php

```diff
@@ -92,6 +95,21 @@ public function addLink(SpanContextInterface $context, iterable $attributes = []
         return $this;
     }
 
+    public function addEvent(string $name, int $timestamp = null, iterable $attributes = []): SpanBuilderInterface
+    {
+        $this->events[] = new Event(
+            $name, 
+            $timestamp == null ? microtime(true) * 1e9 : $timestamp,
+            $this->tracerSharedState
+                ->getSpanLimits()
+                ->getLinkAttributesFactory()
```

Comment:
Based on [SpanLimits.php](https://github.com/open-telemetry/opentelemetry-php/blob/17ad56a3f3a64481b6764bb46b5584d7bbc5b138/src/SDK/Trace/SpanLimits.php#L28), I think it should be this instead:

```suggestion
                ->getEventAttributesFactory()
```